src/vmod_memcached.c

```diff
@@ -76,7 +76,10 @@ get_memcached(const struct vrt_ctx *ctx, struct vmod_mc_vcl_settings *settings)
 	struct timespec wait;
 
 	CHECK_OBJ_NOTNULL(settings, VMOD_MC_SETTINGS_MAGIC);
-	AN(settings->pool);
+	if (!settings->pool) {
+		VSLb(ctx->vsl, SLT_Error, "Could not connect to memcached.");
+		VRT_handling(ctx, VCL_RET_FAIL);
+	}
```

Comment:
It might be a good idea to also `return (NULL);` and not run into `memcached_pool_fetch` with a `NULL` pool.